Add /usr/share/lua/common to LUA_PATH. We use this directory for Lua modules
that are compatible with Lua 5.1 and newer.

--- a/src/luaconf.h
+++ b/src/luaconf.h
@@ -42,8 +42,10 @@
 #ifdef LUA_ROOT
 #define LUA_JROOT	LUA_ROOT
 #define LUA_RLDIR	LUA_ROOT "/share" LUA_LUADIR
+#define LUA_RLDIR2	LUA_ROOT "/share/lua/common/"
 #define LUA_RCDIR	LUA_ROOT "/" LUA_MULTILIB LUA_LUADIR
 #define LUA_RLPATH	";" LUA_RLDIR "?.lua;" LUA_RLDIR "?/init.lua"
+#define LUA_RLPATH2	";" LUA_RLDIR2 "?.lua;" LUA_RLDIR2 "?/init.lua"
 #define LUA_RCPATH	";" LUA_RCDIR "?.so"
 #else
 #define LUA_JROOT	LUA_LROOT
@@ -58,7 +60,7 @@
 #define LUA_LCPATH1	";" LUA_LCDIR "?.so"
 #define LUA_LCPATH2	";" LUA_LCDIR "loadall.so"
 
-#define LUA_PATH_DEFAULT	"./?.lua" LUA_JPATH LUA_LLPATH LUA_RLPATH
+#define LUA_PATH_DEFAULT	"./?.lua" LUA_JPATH LUA_LLPATH LUA_RLPATH LUA_RLPATH2
 #define LUA_CPATH_DEFAULT	"./?.so" LUA_LCPATH1 LUA_RCPATH LUA_LCPATH2
 #endif
 
